recipes/triqs_dft_tools/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+
+# Openmpi Specific environment setup - Cf. https://github.com/conda-forge/libnetcdf-feedstock/pull/80
+export OMPI_MCA_btl=self,tcp
+export OMPI_MCA_plm=isolated
+export OMPI_MCA_rmaps_base_oversubscribe=yes
+export OMPI_MCA_btl_vader_single_copy_mechanism=none
+mpiexec="mpiexec --allow-run-as-root"
+
+source $PREFIX/share/triqsvars.sh
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    ..
```

Comment:
This cmake call should also have set the build type to release as well.

Reply:
I now specify the build type explicitly

Reply:
I now specify the build type explicitly